web/src/components/troubleshoot/TroubleshootContainer.tsx

```diff
@@ -20,7 +20,7 @@ type State = {
   isGeneratingBundle: false;
   generateBundleErrMsg: string;
   loading: boolean;
-  bundleAnalysisProgress: Object;
+  bundleAnalysisProgress?: SupportBundleProgress;
```

Comment:
i added this `SupportBundleProgress` type based off of the go types and just threw it on here since i was already looking at the file.

Reply:
tysm